iroh/src/discovery.rs

```diff
@@ -398,15 +398,15 @@ impl DiscoveryTask {
             }
         };
         let mut on_first_tx = Some(on_first_tx);
-        debug!("discovery: start");
+        debug!("starting");
         loop {
             match stream.next().await {
                 Some(Ok(r)) => {
                     if r.node_addr.is_empty() {
-                        debug!(provenance = %r.provenance, "discovery: empty address found");
+                        debug!(provenance = %r.provenance, "empty address found");
                         continue;
                     }
-                    debug!(provenance = %r.provenance, addr = ?r.node_addr, "discovery: new address found");
+                    debug!(provenance = %r.provenance, addr = ?r.node_addr, "new address found");
```

Comment:
These prefixes are redundant, this function is already called with a "discovery" instrument span.